core/backend/src/ChannelControl.ts

```diff
@@ -73,105 +73,3 @@ export namespace ChannelControl {
   export const sharedChannelName = "shared";
 }
 
-/** @internal */
-export class ChannelAdmin implements ChannelControl {
```

Comment:
Are we sure that someone isn't using this? I know we had a plan to deprecate most of these first to avoid unintended breaking changes.

Reply:
ChannelControl is the public interface. I can't imagine why or how anyone would use the concrete implementation directly. If they are, they can fix their code. We held up our end of the API contract.